examples/cfd/darcy_physics_informed/download_data.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import h5py
+import numpy as np
+from utils import download_FNO_dataset
+
+download_FNO_dataset("Darcy_241", outdir="datasets/")
+
+filename = "datasets/Darcy_241/piececonst_r241_N1024_smooth1.hdf5"
+
+output_filenames = [
+    "datasets/Darcy_241/train.hdf5",
+    "datasets/Darcy_241/validation.hdf5",
+]
+
+# split_percentage = [80, 20]
+split_percentage = [10, 10]
```

Comment:
Does this mean we are only using 20% of the data?

Reply:
Yes, to simulate sparse data training.